lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -47,9 +49,11 @@ public static MarkupContent getMarkupDescription(Parameter parameter) {
 						+ "Expression: " + parameter.getExpression() + LINE_BREAK + "Default Value: " + parameter.getDefaultValue()
 						+ LINE_BREAK + parameter.getDescription());
 	}
+	
+
 
 	public static List<Parameter> collectPluginConfigurationParameters(IPositionRequest request,
```

Comment:
@mickaelistria Maybe this function should now be modified to return a `List<MojoParameter>`? Alternatively, I could make a `collectPluginConfigurationMojoParameters` method?